file-formats/aifr/examples/z_aff_example_aifr.aifr.json

```diff
@@ -6,6 +6,7 @@
     "abapLanguageVersion": "cloudDevelopment"
   },
   "generalInformation":{
-      "namespace": "AIFNS"
+      "namespace": "AIFNS",
+      "aifRecipient": "REC_WS_INB"
```

Comment:
The prefix "AIF" has also been used in other places (like for AIFN for the namespace):
https://github.com/SAP/abap-file-formats/blob/942e346992be50637c8140da6564ddf75f0ef5d2/file-formats/aifn/examples/z_aff_example_aifn.aifn.json#L9

Therefore, I guess it is fine here, too.

Reply:
We refer to the development object: Namespace (AIFN). The namespace (AIFN) contains the field AIF namespace, here we basically convert the name of the namespace (AIFN) into our own AIF namespace. This is necessary due to historical reasons.
In this case it is the namespace and not the internal AIF namespace which needs to be assigned to the recipient, so the prefix would not be correct.

Reply:
Oh, I see. Sorry, my comment was misleading. 

My comment should have been related to field `aifRecipient` (not to field `namespace`). I just wanted to state, that it is most probably fine to use "aif" as prefix for this field as you did it for `aifNamespace` in AIFN.

Reply:
I agree with you. AIF is often used in the editors of this context. Therefore, it is fine for me here as well.

Reply:
Fine for me... I resolve this conversation